service/templates/admin.html

```diff
@@ -1351,11 +1351,22 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 				<md-card-title>
 					<i class="fa fa-refresh"></i>
 					<md-card-title-text>
-						<span>Restart Tasks</span>
-						<span class="md-subhead">Restart failed tasks that ran between two times</span>
+						<span>Restart Tasks/Commit Queue Versions</span>
+						<span class="md-subhead">Restart failed tasks or versions that ran between two times</span>
```

Comment:
Let's add "commit queue" to this description to make it clear it's not used for other things